CONTRIBUTING.md

```diff
@@ -92,5 +92,5 @@ The following contributions are accepted:
 
 **Legend**
 * ✔️: Contributions welcomed!
-* ✓: Contributions welcomed, but must be discussed first
+* ✓: Contributions are welcomed, but must be discussed first
```

Comment:
These two lines are redundant, so the first should probably be removed in favor of the second.

Reply:
Hey! @spenserblack, I don't think they are redundant, please look carefully below given screenshot or visit [CONTRIBUTING.md](/CONTRIBUTING.md).


   ![capture](https://user-images.githubusercontent.com/69510006/153883343-4052282c-c6c9-4c7c-881b-521c1f6724dd.png)

Reply:
Oh, nevermind, ignore my comment. You are correct.

Reply:
That's fine, we are in an open-source community for open-source contributions, with open-mind questions, answers, support, learning, etc.